source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -130,9 +130,21 @@ export class StringWithSubstitutions {
                     break;
                 }
             }
-
             if (matchedInput) {
-                this._processed = this._processed.replace(matches[0], matchedInput.value ? matchedInput.value : "");
+                var valueForReplace = "";
+                if (matchedInput.value)
+                {
```

Comment:
Open curly bracket on the same line as if statement

Reply:
fixed